sdk/messaging_servicebus/examples/service_bus00.rs

```diff
@@ -1,65 +1,43 @@
-// use azure_messaging_servicebus::prelude::*;
-// use chrono::Duration;
-use std::error::Error;
+use azure_messaging_servicebus::prelude::*;
 
-// async fn send(
-//     s: String,
-//     service_bus_namespace: String,
-//     event_hub_name: String,
-//     policy_name: String,
-//     policy_key: String,
-// ) -> Result<(), Box<dyn Error>> {
-//     let mut client = Client::new(
-//         service_bus_namespace.to_owned(),
-//         event_hub_name.to_owned(),
-//         policy_name.to_owned(),
-//         policy_key.to_owned(),
-//     )?;
+#[tokio::main]
+async fn main() {
+    let service_bus_namespace = std::env::var("AZURE_SERVICE_BUS_NAMESPACE")
+        .expect("Please set AZURE_SERVICE_BUS_NAMESPACE env variable first!");
 
-//     println!("before {:?} message send!", s);
-//     match client.send_event(&s, Duration::days(1)).await {
-//         Ok(_) => println!("{:?} message sent!", s),
+    let queue_name =
+        std::env::var("AZURE_QUEUE_NAME").expect("Please set AZURE_QUEUE_NAME env variable first!");
 
-//         Err(error) => println!("{:?} failed to send message", error),
-//     }
+    let policy_name = std::env::var("AZURE_POLICY_NAME")
+        .expect("Please set AZURE_POLICY_NAME env variable first!");
 
-//     Ok(())
-// }
+    let policy_key =
+        std::env::var("AZURE_POLICY_KEY").expect("Please set AZURE_POLICY_KEY env variable first!");
 
-#[tokio::main]
-async fn main() -> Result<(), Box<dyn Error>> {
-    // // First we retrieve the account name and master key from environment variables.
-    // // We expect master keys (ie, not resource constrained)
-    // let service_bus_namespace = std::env::var("AZURE_SERVICE_BUS_NAMESPACE")
-    //     .expect("Set env variable AZURE_SERVICE_BUS_NAMESPACE first!");
-    // let event_hub_name = std::env::var("AZURE_EVENT_HUB_NAME")
-    //     .expect("Set env variable AZURE_EVENT_HUB_NAME first!");
-    // let policy_name =
-    //     std::env::var("AZURE_POLICY_NAME").expect("Set env variable AZURE_POLICY_NAME first!");
-    // let policy_key =
-    //     std::env::var("AZURE_POLICY_KEY").expect("Set env variable AZURE_POLICY_KEY first!");
+    let http_client = azure_core::new_http_client();
+
+    let mut client = Client::new(
+        http_client,
+        service_bus_namespace,
+        queue_name,
+        policy_name,
+        policy_key,
+    )
+    .expect("Failed to create client");
+
+    let message_to_send = "hello, world!";
 
-    // let messages = vec![
-    //     "These", "are", "useless", "messages", "provided", "for", "free", "with", "love",
-    // ];
-    // println!(
-    //     "Sending the following messages: {:?}. \
-    //      Please note they will be sent out of order!",
-    //     messages
-    // );
+    client
```

Comment:
it would be great to have 
1. multi-thread example using rust channels, where one thread sends msg to the queue, and another thread recieves the msg from the queue. 
2. multi sender-receiver where one sender sends msg to multiple queues, and mutiple recivers are able to subscribe to different queues. 

what do you think?

Reply:
Hmmm, I don't know if we should block the PR on examples.

Also, I should note: the Azure Service Bus API does not include the concept of "subscribing".